packages/plugins/i18n/admin/src/components/CMEditViewLocalePicker.tsx

```diff
@@ -82,7 +82,12 @@ const CMEditViewLocalePicker = ({
       return;
     }
 
-    if (status === 'did-not-create-locale') {
+    /**
+     * TODO: if D&P is not enabled, then the status will always say there's no locale so
+     * we also should check there's no ID incase. This logic will be removed in V5 when
+     * we _always_ have D&P.
```

Comment:
this behaviour probably will have to be in V5 as we will have disabled D&P on content types too!

Reply:
Probably because I lack context here, but I do not quite understand the if case, and how D&P is interacting with locale creation. 

From what  I understood the status is `did-not-create-locale` when that locale doesnt exist, but also when D&P is not enabled?

should the status be set to that value when there is an actual locale?

Also, would it be enough just to check if the id of the locale exists or not?

Reply:
Yeah I can probably make a better solution for this now we know D&P isn't always enabled. Because it doesn't make a great deal of sense & there will be new hooks as well to work with...

Have you QA'd the fix? 👀

Reply:
nope! going to tesst it in 5 min

Reply:
functionality seems to be working ! can swap the locale with a content type with disabled D&P

Reply:
Ey, oh, let's merge!